test/lists.js

```diff
@@ -1,72 +1,272 @@
-const chai = require('chai')
-const expect = chai.expect
-
+const { expect } = require('chai')
 const Hubspot = require('..')
-const hubspot = new Hubspot({ apiKey: 'demo' })
+const fakeHubspotApi = require('./helpers/fake_hubspot_api')
+const { createTestContact, deleteTestContact } = require('./helpers/factories')
+
+describe('lists', function() {
+  const hubspot = new Hubspot({
+    accessToken: process.env.ACCESS_TOKEN || 'some-fake-token',
+  })
+  const listProperties = { name: 'Test list name' }
+  const createTestList = () => hubspot.lists.create(listProperties)
+  const deleteTestList = listId => hubspot.lists.delete(listId)
+
+  describe('get', function() {
+    const listsEndpoint = {
+      path: '/contacts/v1/lists',
+      response: { lists: [] },
+    }
+    fakeHubspotApi.setupServer({ getEndpoints: [listsEndpoint] })
 
-describe('Lists', function() {
-  describe('Get Lists', function() {
-    it('Should return contact lists', function() {
+    it('should return contact lists', function() {
       return hubspot.lists.get().then(data => {
         expect(data).to.be.a('object')
         expect(data.lists).to.be.a('array')
       })
     })
   })
 
-  describe('Get List by Id', function() {
-    it('Should return one contact in a list', function() {
-      return hubspot.lists.getOne(1).then(data => {
-        expect(data).to.be.a('object')
-        expect(data).to.have.a.property('name')
+  describe('getOne', function() {
+    describe('when passed a listId', function() {
+      let listId = 123
+      const listEndpoint = {
+        path: `/contacts/v1/lists/${listId}`,
+        response: listProperties,
+      }
+      fakeHubspotApi.setupServer({ getEndpoints: [listEndpoint] })
+
+      before(function() {
+        if (process.env.NOCK_OFF) {
+          return createTestList(listProperties).then(
+            data => (listId = data.listId),
+          )
+        }
+      })
+      after(function() {
+        if (process.env.NOCK_OFF) {
+          return deleteTestList(listId)
+        }
+      })
+
+      it('should return one contact in a list', function() {
```

Comment:
I think this returns a ContactList rather than a Contact in a specific ContactList. Thoughts on updating this to read "returns a ContactList"?

Reply:
Updated! Thanks.